config/deploy/dev.rb

```diff
@@ -1,4 +1,4 @@
-set :bundle_without, %w[sqlite production test].join(' ')
+set :bundle_without, %w[production].join(' ')
```

Comment:
Does this mean `pg` is not available on dev?

Reply:
yes. we should probably switch -dev to use RAILS_ENV=production too. I only use dev to test the index. what are you using -dev for, if anything?

Reply:
Just to randomly test application code. I'm 👍 for the change.